.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ["bug", "triage"]
```

Comment:
Don't think you automatically want to label things as bug.

Reply:
I do ... hence the triage so we have to check it first ... easy to find unreviewed bugs